src/components/views/room_settings/RoomPublishSetting.tsx

```diff
@@ -50,7 +59,7 @@ export default class RoomPublishSetting extends React.PureComponent<IProps, ISta
         client
             .setRoomDirectoryVisibility(this.props.roomId, newValue ? Visibility.Public : Visibility.Private)
             .catch(() => {
-                // Roll back the local echo on the change
```

Comment:
This comment was useful and accurate though, so should probably stay.

Reply:
sure, adding that back

Reply:
Sorry, one more thing: this comment is now above the wrong line: it should be one down, above the line it pertains to.

Reply:
okay, i will change

Reply:
done